tests/modes/apps/extensions/test_extension_remote.py

```diff
@@ -17,172 +16,73 @@ class TestExtensionRemote:
     def remote(self) -> ExtensionRemote:
         return ExtensionRemote("https://github.com/Ulauncher/ulauncher-timer")
 
-    def test_valid_urls_ext_id(self) -> None:
-        assert ExtensionRemote("https://host.tld/user/repo").ext_id == "tld.host.user.repo"
-        assert ExtensionRemote("http://host/user/repo").ext_id == "host.user.repo"
-        assert ExtensionRemote("https://host.org/user/repo.git").ext_id == "org.host.user.repo.git"
-        assert ExtensionRemote("http://host/user/repo.git").ext_id == "host.user.repo.git"
-        assert ExtensionRemote("git@host.com:user/repo").ext_id == "com.host.user.repo"
-        # verify sanitizing github/gitlab/codeberg urls, but leave all others
-        assert ExtensionRemote("https://github.com/user/repo/tree/HEAD").ext_id == "com.github.user.repo"
-        assert ExtensionRemote("https://gitlab.com/user/repo.git").ext_id == "com.gitlab.user.repo"
-        assert ExtensionRemote("https://other.host/a/b/c/d").ext_id == "host.other.a.b.c.d"
-
     def test_invalid_url(self) -> None:
         with pytest.raises(InvalidExtensionRecoverableError):
             ExtensionRemote("INVALID URL")
 
-    def test_get_download_url(self) -> None:
-        assert (
-            ExtensionRemote("https://github.com/user/repo")._get_download_url("master")
-            == "https://github.com/user/repo/archive/master.tar.gz"
-        )
-        assert (
-            ExtensionRemote("https://gitlab.com/user/repo")._get_download_url("master")
-            == "https://gitlab.com/user/repo/-/archive/master/repo-master.tar.gz"
-        )
-
 
 class TestParseExtensionUrl:
-    @patch("ulauncher.modes.extensions.extension_remote.which")
-    def test_https_url(self, mock_which: MagicMock) -> None:
-        mock_which.return_value = "/usr/bin/git"
+    def test_https_url(self) -> None:
         result = parse_extension_url("https://example.com/user/repo")
+        assert result.remote_url == "https://example.com/user/repo"
```

Comment:
Intentionally changed to test only the affected props, not the full output. Otherwise any change will break all tests.